assignment/fish-assignment.Rmd

```diff
@@ -62,46 +89,29 @@ build a tidy table with the desired columns.
  
 ```
 
-## Task 2: Mapping the Area table to marine regions
-
-In order to replicate the collapse of Atlantic Cod, 
-we need to be able to map area table from the Ram database to the marine regions. 
-
-*As an aside, this database is unclear what kind of areas the `area` table is using, they do not appear to be LMEs, EEZs, or other obvious marine region classification. Regardless, we will use them to extract the North America cod stocks.*
+I ended up with a table that had the following column names (some of which I needed to rename
+in the process!):  `country`, `areaname`, `ssb_unit`, `TC_unit`, 
+`scientificname`, `commonname`, `year`, `ssb`, and `TC`. Your columns might be different
+just make sure you contain the info listed above!
 
-Write code to pull all marine areas (listed in `ram$area`) that contain a certain substring
-in their name -- ex. "Georges Bank". 
-Hint: you want want to consider functions `filter()` or `grep()`
+## Task 2: Subsetting our data by regional id and species name 
 
-```{r}
-
-```
+In order to replicate the collapse of Atlantic Cod, we are interested in the 
+data from Atlantic Cod harvests in Canada. 
```

Comment:
Yeah, this section is definitely confusing.  In order to get the y-axis numbers to add up to approximately the same counts as in the millennium  ecosystem assessment, I think you basically need Canada, USA, and the two international waters shared by US and Canada (Georges Banks) that collectively make up the American continent side of the North Atlantic.  Canada is a close proxy for this though since the bulk is in Canadian waters.

I agree about dropping `grep`, since we haven't covered it and it's better they use region names.

Also this exercise raises the question of how to select `Atlantic cod`.  RAM common names are inconsistent, e.g.  `Atlantic Cod` vs `Atlantic cod`, though scientific names are more reliable.  The database actually includes TSN (tax species id) which are more 'typo-proof' than scientific names, so a more natural use as a database key, but also more opaque.